working/augmentation-libraries/feature-specification.md

```diff
@@ -518,14 +530,14 @@ More specifically:
     necessary to ensure that macros running after signatures are known can't
     change the signature of a declaration.*
 
-    **TODO: What if the augmenting variable doesn't have an initializer?**
-
 It is a compile-time error if:
 
 *   The original and augmenting declarations do not have the same type.
 
 *   An augmenting declaration uses `augment super` when the original declaration
-    is marked `external`.
+    has no concrete implementation. It **is** allowed to call `augment super`
+    when augmenting an external declaration _if_ an implementation has been
+    filled in by a compiler or other external source.
```

Comment:
That if is platform/implementation-specific. It's probably better to just say that unconditionally that all `external` declarations are treated as having implementations.